src/store/offers/actions.js

```diff
@@ -1,17 +1,30 @@
+import { signerSelector } from 'src/store'
 import { daoByType } from '../daos/selectors'
+import { JOB_STATE_CREATED } from "./../../models/app/JobStateModel"
 
 export const OFFERS_CLEAR = 'offers/clear'
 export const OFFERS_SAVE = 'offers/save'
 
 // Should be called only once
-export const initJobOffers = () => async (/*dispatch, getState*/) => {
-  // const state = getState()
-  // TODO aevalyakin subscribe on events here
+export const initJobOffers = () => async ( dispatch) => {
+  await dispatch(reloadJobsOffers())
 }
 
-export const reloadJobsOffers = ({ jobId }) => async (dispatch, getState) => {
+export const reloadJobsOffers = () => async (dispatch, getState) => {
   const state = getState()
   const jobDataProviderDAO = daoByType('JobsDataProvider')(state)
-  const offers = await jobDataProviderDAO.getJobOffers(jobId)
-  dispatch({ type: OFFERS_SAVE, jobId, offers })
+  const signer = signerSelector()(state)
+  if (state.jobs && state.jobs.list) {
+    for (let job of state.jobs.list) {
+      if (job.state===JOB_STATE_CREATED) 
+      {setTimeout(async () => {
+        const allOffers = await jobDataProviderDAO.getJobOffers(job.id)
+        //Filtering only my offers
+        const offers=allOffers.filter((item) => item.worker===signer.address )
+        if (offers.length>0)
+        {dispatch({ type: OFFERS_SAVE, jobId: job.id, offers  })}
```

Comment:
if (offers.length>0) {
--dispatch({ type: OFFERS_SAVE, jobId: job.id, offers  })
}

not 
if (offers.length>0)
{dispatch({ type: OFFERS_SAVE, jobId: job.id, offers  })}